internal/pkg/template/template.go

```diff
@@ -37,6 +37,7 @@ const (
 	ReadmeFile           = "README.md"
 	SrcDirectory         = "src"
 	TestsDirectory       = "tests"
+	InstanceId           = "instance-id"
```

Comment:
Mohol by si to spravi prosim nejake jasnejsie `InstanceIdForTest`? , alebo pridat komentar.

Reply:
Este mi napadlo, ze by si to mohol pouzit v e2e testoch ako monika, v `name` aby sme si boli isty, ze to funguje (`InstanceIdShort`).

Reply:
jo, pravda 👍

Reply:
https://github.com/keboola/keboola-as-code/pull/894/commits/64e1e8e28d8f4394f1b88ba612faef41fbcf1f33